docs/usage-examples/deleteMany.txt

```diff
@@ -0,0 +1,67 @@
+.. _laravel-delete-many-usage:
+
+=========================
+Delete Multiple Documents
+=========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: delete many, remove, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can delete multiple documents in a collection by calling the ``delete()`` method on an
+Eloquent model or a query builder.
+
+To delete multiple documents, pass a query filter to the ``where()`` method. Then, delete the
+matching documents by calling the ``delete()`` method.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database
+- Deletes documents from the ``movies`` collection that match a query filter
+
+The example calls the following methods on the ``Movie`` model:
+
+- ``where()``: matches documents in which the value of the ``year`` field is less than or
+  equal to ``1910``
+- ``delete()``: deletes the retrieved documents
```

Comment:
Suggestion:

I think it could be helpful to add some information about the value printed even if it seems self-explanatory. E.g.
"The delete() method returns the number of documents successfully deleted"